### PR TITLE
Add Image component, modify TableView component

### PR DESCRIPTION
**Pull request by mvn-binhnguyen-hn**
# Modify some components

## What are you adding?
- Adding `Image` to handle error image links
- Adding `submitOnChange` prop to prevent auto submitting when submit button is hidden
- Adding `View` button at TableView component

### CODE REVIEW

File: src/components/atoms/Image/Image.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  useEffect(() => {
+    setUrl(src);
+  }, [src]);
+
+  return (
+    <img
+      {...rest}
+      src={url}
+      alt={rest.alt}
+      onError={() => setUrl(defaultImg ?? notFoundImage)}
```
Comment (jacksonjp) [inline src/components/atoms/Image/Image.tsx:29]: maybe lets add onLoad to show a loading spinner even

File: src/components/molecules/PageFilter/PageFilter.tsx [hunk 2]
```diff
@@ -63,6 +68,7 @@ const PageFilter = <T extends {}>({
   columns = 4,
   showResetButton,
   showSubmitButton,
+  submitOnChange,
```
Comment (jacksonjp) [inline src/components/molecules/PageFilter/PageFilter.tsx:71]: Maybe we should default it to true so we dont change the current behaviour, so if we can explicitly disable if its not required

File: src/components/molecules/TableView/TableView.tsx [hunk 3]
```diff
@@ -118,6 +121,15 @@ const TableView = <T extends {}>({
           className={styles.actions}
           render={(_, record) => (
             <Space size="middle">
+              {!!onView && (
```
Comment (jacksonjp) [inline src/components/molecules/TableView/TableView.tsx:124]: nice

File: src/features/localization/helpers/localization.helpers.ts [hunk 4]
```diff
@@ -32,6 +32,10 @@ export const changeLanguage = (locale: string) => {
   updateLocalization();
 };
 
+export const getCurrentLanguage = () => {
```
Comment (jacksonjp) [inline src/features/localization/helpers/localization.helpers.ts:35]: not sure we are using this yet! Also not sure if languages[0] will always return the current language.

**Review by jacksonjp — CHANGES_REQUESTED** [review]
Thanks a lot! Leaving some comments!

File: src/features/localization/helpers/localization.helpers.ts (hunk 4 above)
Comment (mvn-binhnguyen-hn) [reply src/features/localization/helpers/localization.helpers.ts:35]: Fixed. https://www.i18next.com/overview/api#language

**Review by jacksonjp — APPROVED** [review]
Looks good to me :)

**Pull request merged**